runtime/runtime.go

```diff
@@ -211,6 +211,9 @@ type Params struct {
 	DiskStorage *disk.Options
 
 	DistributedTracingOpts tracing.Options
+
+	// Check if default Addr is set or the user has changed it.
+	AddrChanged bool
```

Comment:
Nit: Can we rename `AddrChanged`  to `AddrSetByUser` or  something similar that conveys that user set the value.

Reply:
Yes indeed this makes a lot more sense, will update it :+1: